src/BuildServer/BuildServerRequests.cs

```diff
@@ -0,0 +1,24 @@
+﻿// --------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+// --------------------------------------------------------------------------------------------
+
+using Newtonsoft.Json;
+
+namespace Microsoft.Oryx.BuildServer
+{
+    public class BuildServerRequests
+    {
+        [JsonProperty(PropertyName = "command")]
+        public string command { get; set; }
+
+        [JsonProperty(PropertyName = "id")]
+        public string id { get; set; }
```

Comment:
How about id -> buildRequestId

Reply:
This file is actually not used. I'm removing this file. Thanks Stella.